src/ChainSafe.Gaming/Web3/Core/Build/Web3Builder.cs

```diff
@@ -63,14 +70,19 @@ public Web3Builder Configure(ConfigureServicesDelegate configureMethod)
             return this;
         }
 
+        /// <summary>
+        /// Build <see cref="Web3"/> object using the settings provided by this Web3Builder object.
+        /// </summary>
+        /// <returns><see cref="Web3"/> object.</returns>
         public async ValueTask<Web3> BuildAsync()
```

Comment:
Building web3 is failing on Unity with runtime error `A suitable constructor for type 'ChainSafe.Gaming.Evm.ChainRegistryProvider' could not be located. Ensure the type is concrete and services are registered for all parameters of a public constructor`

Reply:
Fixed